docs/codesnippets/import_chrome_session.py

```diff
@@ -0,0 +1,91 @@
+# https://n8henrie.com/2013/11/use-chromes-cookies-for-easier-downloading-with-python-requests/
+
+from argparse import ArgumentParser
+from glob import glob
+from os.path import expanduser
+from platform import system
+from sqlite3 import OperationalError, connect
+import win32crypt
```

Comment:
What is `win32crypt`?

Reply:
It's a module that is used for cryptographic operations. Chrome cookies are encrypted when they are stored and it's required to retrieve the actual value of the cookies

Reply:
I can't find a package for `win32crypt` in ubuntu 22.04

Reply:
I realized this issue and I'm working on a way of replacing it with PyCryptodome or at least making a check for Windows vs Linux

Reply:
Hi
Any Update on the linux/ubunutu version?

Reply:
@OMGhozlan

Reply:
@patilaum not realy :'(